### ISSUE TITLE
Database_exist not working in python 3.11

### ISSUE THREAD

**Issue by pmithrandir** [kvesteri/sqlalchemy-utils#716]
Hello,

I tried to migrate to python 3.11.5 from python 3.9.

When executing the following code, I get an error.
```
from sqlalchemy_utils import database_exists
test = database_exists("postgresql://postgres:pass@localhost:5432/test")
```
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe9 in position 103: invalid continuation byte

My env : 
sql alchemy 1.4 or 2.0
psycopg2 vers 2.9.7
windows
postgres PostgreSQL 12.13, compiled by Visual C++ build 1914, 64-bit

I can't really understand what is the issue.

Do you have any idea ?
Regards,
Pierre

**Comment by n0nuser** [comment]
Hello!

Same error here: `UnicodeDecodeError: 'utf-8' codec can't decode byte 0xab in position 96: invalid start byte` when executing `database_exists` function.

My colleagues were having these issues with Python 3.11.8 and I didn't: I was using Python 3.10.11. I then upgraded to double check to their version, and then I got the error.

These are the package versions we have:

* `sqlalchemy==2.0.27`
* `sqlalchemy-utils==0.41.1`
* `psycopg2-binary==2.9.9`

For the time being, my team and I are rolling back to Python 3.10.8 until this is stable for newer versions.
